addons/l10n_tr_nilvera_edespatch/models/res_partner.py

```diff
@@ -0,0 +1,31 @@
+from odoo import _, models
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    def _l10n_tr_nilvera_validate_partner_details(self):
+        error_messages = {}
+        for record in self:
+            required_fields = {
+                "street": record.street,
```

Comment:
```suggestion
                _("street"): record.street,
```
apply the same below